src/spatial_grid.jl

```diff
@@ -1,35 +1,21 @@
 """
-` Rasterize a points cloud in 2D`
+    rasterize_points(cloud::PointCloud, dx::AbstractFloat)
+    rasterize_points(cloud::Matrix{AbstractFloat}, dx::AbstractFloat)
```

Comment:
Chris - yeah sometimes they are the positions stored as a PointCloud. 

I will use `cloud::Matrix{<:AbstractFloat}` for the docs.